zerver/views/documentation.py

```diff
@@ -152,20 +153,24 @@ def get_context_data(self, **kwargs: Any) -> Dict[str, Any]:
             documentation_article.article_path
         ):
             # Absolute path case
-            article_path = documentation_article.article_path
+            article_full_path = documentation_article.article_path
```

Comment:
I renamed this because of the overlap with the `DocumentationArticle` dataclass field `article_path`. This could be moved to a separate commit for clarity, but I wondered if there was a better variable name to use. I wanted something that wouldn't overlap with `article_path` when grepping.

Reply:
Yeah, that's reasonable. I think `article_absolute_path` or even just `absolute_path` would be pretty good here.

Reply:
I went with `article_absolute_path` to distinguish it from a few other `absolute_path` uses in the code base.